robotpy_installer/sshcontroller.py

```diff
@@ -50,10 +50,13 @@ def ssh_exec_commands(self, commands, existing_connection=False):
         if not existing_connection:
             self.ssh_connect()
 
-        _, stdout, _ = self.client.exec_command(commands)
+        _, stdout, stderr = self.client.exec_command(commands)
 
         for line in iter(stdout.readline, ""):
             print(line, end="")
+        for line in iter(stderr.readLine, ""):
+            # Print stderr, using red color to help distinguish
+            print("\033[91m" + line + "\033[0m", end="")
```

Comment:
It would be good to use something like colorama to do this cross-platform, otherwise the output is going to look really weird on some platforms.

Reply:
I was thinking about it, but didn't want to add an extra dependency specifically for just coloring the text. I definitely can use it if necessary, just seems like overkill for a single spot.